### PR TITLE
SAT Sprint-2 Development

### PR DESCRIPTION
**Pull request by deepakmulamalla**
SAT Sprint-2 Development

User Stories:
ESA - 614,
ESA - 1316,
ESA - 1408,
ESA-1674.

### CODE REVIEW

File: force-app/main/default/objects/Case/fields/Rationale__c.field-meta.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <label>Rationale</label>
     <length>32768</length>
     <trackFeedHistory>false</trackFeedHistory>
-    <trackHistory>false</trackHistory>
```
Comment (cgiSunilM) [inline force-app/main/default/objects/Case/fields/Rationale__c.field-meta.xml:8]: Any particular reason to remove tracking history that was there earlier.

**Review by cgiSunilM — CHANGES_REQUESTED** [review]

File: force-app/main/default/lwc/esa_LC_getApprovalHistory/esa_LC_getApprovalHistory.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  
+  rationalTextLabel = ESA_label_rationalText;
+  saApprovalRequestFormatData = [];
+
+  connectedCallback() {
+    this.fetchItems();
+  }
+
+  async fetchItems() {
+    console.log('called'+this.phn);
```
[the remark this answers is not in the document]
Comment (deepakmulamalla) [reply force-app/main/default/lwc/esa_LC_getApprovalHistory/esa_LC_getApprovalHistory.js:40]: Console statement has been removed and changes has been commited.

File: force-app/main/default/lwc/esa_LC_getApprovalHistory/esa_LC_getApprovalHistory.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+          let item = {};
+          item['description'] = rec.specialItem.itemDescription;
+          item['dinrdp'] = this.convertDINPIN(rec.specAuthType, rec.specialItem.din || rec.specialItem.rdp);
+          item['specAuthType'] = this.convertSAType(rec.specAuthType);
+          item['effectiveDate'] = rec.effectiveDate;
+          item['terminationDate'] = rec.terminationDate;
+          item['practId'] = rec.saRequester.practId;
+          item['practIdRef'] = rec.saRequester.practIdRef;
+          item['excludedPlans'] = "";
+          rec.excludedPlans.forEach(ep => {
```
[the remark this answers is not in the document]
Comment (deepakmulamalla) [reply force-app/main/default/lwc/esa_LC_getApprovalHistory/esa_LC_getApprovalHistory.js:82]: Changes has been removed and changes has been committed.

File: force-app/main/default/objects/Case/fields/Rationale__c.field-meta.xml (hunk 1 above)
Comment (deepakmulamalla) [reply force-app/main/default/objects/Case/fields/Rationale__c.field-meta.xml:8]: Hi Sunil, here we are enabling the field tracking that was not there earlier as per user story ESA-1316.

**Review by cgiSunilM — APPROVED** [review]
So far it looks good to me! But needs improvement.

**Pull request merged**